src/lib/components/ExpertPhotoDataCard.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
```

Comment:
can we use a composition of 2 data cards for experts?
1 - card to display expert photo (used on main for example)
2- card to display expert description (used on main expert popover)

here we can use the composition of 1 and 2 ..

Reply:
done